Unify chat with us  widget design across all pages

jai3546 opened the pull request:
- Add unified chat widget styles to style.css as single source of truth  
- Remove duplicate/conflicting chat styles from faq.css  
- Include style.css and script.js in faq.html and homestays.html  
- Add enhanced hover effects and responsive design for chat buttons  
- Fix chat header text consistency ('Chat With Us')  
- Modify server.js to run without database temporarily  
- Ensure consistent chat widget appearance across index, faq, and homestays pages  

# 🔀 Pull Request

## 📌 Issue Reference  
Closes #435  
> 🐛 Issue #435 was identified and created by me to document the bug before this fix.  

---

## 📝 Summary  
This PR resolves inconsistent chat widget styling across different pages.  
- Unified styles under `style.css` (removed duplicate FAQ overrides).  
- Applied hover effects and responsive layout for buttons.  
- Standardized header text ("Chat With Us").  
- Updated `faq.html` and `homestays.html` to use correct assets.  
- Modified `server.js` to allow running without a database temporarily.  

---

## 📸 Screenshots (if applicable)  
**Before (FAQ page – inconsistent):**  
![Before](https://github.com/user-attachments/assets/55005d97-c892-467b-b10c-b88eb16c5ec8)  

**After (Unified styles – consistent):**  
![After](https://github.com/user-attachments/assets/215c6a6b-0d35-4682-a205-7980f0df09c2)  

---

## ✅ Checklist  
- [x] My code follows the project's coding conventions  
- [x] I have tested all impacted features  
- [x] I have updated or added necessary documentation  

---

## 🔗 Related Issues / PRs  
- Issue: #435 (identified and created by me)  

---

## 🏅 Open Source Program Participation  
**Open Source Connect India**  

---

## 💬 Additional Notes  
This PR ensures a unified, responsive, and professional chat widget experience across all pages.

jai3546 committed:
Unify chat widget design across all pages

- Add unified chat widget styles to style.css as single source of truth
- Remove duplicate/conflicting chat styles from faq.css
- Include style.css and script.js in faq.html and homestays.html
- Add enhanced hover effects and responsive design for chat buttons
- Fix chat header text consistency ('Chat With Us')
- Modify server.js to run without database temporarily
- Ensure consistent chat widget appearance across index, faq, and homestays pages

jai3546 committed:
Merge branch 'main' into feature/fixing-chat-with-us-infaq

Nayanika1402 commented:
@jai3546 Send pr in dev branch.

jai3546 committed:
Merge branch 'dev' into feature/fixing-chat-with-us-infaq

jai3546 commented:
> @jai3546 Send pr in dev branch.

Check Now @Nayanika1402

Nayanika1402 commented:
@jai3546 The FAQ page looks like this:
<img width="1919" height="1079" alt="image" src="https://github.com/user-attachments/assets/6df1e7b9-7e68-4a26-b8ed-158e6cf709e5" />
Solve this issue.

jai3546 committed:
Fix FAQ page click functionality and navbar design

- Fixed FAQ accordion functionality with proper click handlers
- Removed duplicate head sections from faq.html
- Removed login button from navbar as requested
- Fixed navbar structure and mobile menu
- Improved FAQ JavaScript with DOMContentLoaded wrapper
- Added proper accordion behavior (only one answer open at a time)
- Fixed arrow rotation using text content (▼/▲)
- Minor CSS improvements for FAQ answer styling
- Removed conflicting script.js reference

jai3546 committed:
Merge and fix FAQ functionality conflicts

- Resolved merge conflicts in faq.html
- Fixed FAQ click functionality with proper accordion behavior
- Removed login button from navbar as requested
- Improved FAQ JavaScript with DOMContentLoaded wrapper

jai3546 committed:
Merge branch 'Nayanika1402:main' into feature/fixing-chat-with-us-infaq

jai3546 committed:
fixed duplicated faqs in faqs.html

Nayanika1402 commented:
@jai3546 Resolve the conflicts.

jai3546 committed:
Merge branch 'dev' into feature/fixing-chat-with-us-infaq

jai3546 committed:
fixed minor ui issues and also fixed chat with us unified across all the pages.

jai3546 committed:
unified ui and fixed minor conflict

jai3546 committed:
Merge branch 'dev' into feature/fixing-chat-with-us-infaq

Nayanika1402 commented:
<img width="1919" height="1079" alt="image" src="https://github.com/user-attachments/assets/f9034f91-e6e2-47c1-9646-48615cfe6dc4" />
This problem in ui is in almost evry page the options of sending deleting the message has css problems.Kindly solve it @jai3546

jai3546 committed:
fixed contact chat with us

jai3546 commented:
<img width="1906" height="966" alt="Screenshot 2025-09-02 001450" src="https://github.com/user-attachments/assets/4c8869af-d3e2-4304-aa57-58ffbad92896" />
Check now @Nayanika1402 .
If any changes let me know!

jai3546 commented:
@Nayanika1402 Due to exams i didn't work on it,now I will fix it

jai3546 committed:
Merge branch 'dev' into feature/fixing-chat-with-us-infaq
